<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1" colspan="3">
     <widget class="QLabel" name="label_time">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>300</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; color:#7f7f7f;&quot;&gt;--:--&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextEditable</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="button_start">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 138, 138);
</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="button_reset">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="4">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionTime_Setting"/>
    <addaction name="actionAlarm"/>
   </widget>
   <widget class="QMenu" name="menuTo_Do">
    <property name="title">
     <string>To Do</string>
    </property>
    <addaction name="actionShow_List"/>
   </widget>
   <widget class="QMenu" name="menuStatistic">
    <property name="title">
     <string>Statistic</string>
    </property>
    <addaction name="actionShow_Data"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuMod">
    <property name="title">
     <string>Mod</string>
    </property>
    <addaction name="actionFace_Detect"/>
   </widget>
   <addaction name="menusetting"/>
   <addaction name="menuTo_Do"/>
   <addaction name="menuStatistic"/>
   <addaction name="menuInfo"/>
   <addaction name="menuMod"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTime_Setting">
   <property name="text">
    <string>Time Setting</string>
   </property>
  </action>
  <action name="actionAlarm">
   <property name="text">
    <string>Alarm</string>
   </property>
  </action>
  <action name="actionShow_List">
   <property name="text">
    <string>Show List</string>
   </property>
  </action>
  <action name="actionShow_Data">
   <property name="text">
    <string>Show Data</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About </string>
   </property>
  </action>
  <action name="actionFace_Detect">
   <property name="text">
    <string>Face Detect</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
